packages/eslint-plugin/src/rules/no-empty-styled-expression/index.ts

```diff
@@ -25,9 +25,20 @@ const createNoEmptyStyledExpressionRule =
     return {
       'CallExpression[callee.type="MemberExpression"]': (node: CallExpression) => {
         const { references } = context.getScope();
```

Comment:
If you leave the callee check in, this should be placed after the early return. It will ensure you do less work, by not having to evaluate `getScope()` until it's needed.